src/sagas/repository.ts

```diff
@@ -787,16 +787,19 @@ export function* watchGetDatasetByIdSuccess(): any {
 
 export function* previewData({ payload }: any): any {
```

Comment:
While you're in here, should we rename the method viewData? Or something that doesn't have preview in the name?

Reply:
I think it would be hard to actually remove all of the references to "preview" data. I think I'd rather leave it all as preview rather than changing only some of the references to the new name, especially since this would only be a code-facing change rather than user-facing.